source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -1516,8 +1547,8 @@ Authentication errors
 ~~~~~~~~~~~~~~~~~~~~~
 
 If the authentication handshake fails for a connection, drivers MUST mark the
-server Unknown and clear the server's connection pool. (See
-`Why mark a server Unknown after an auth error?`_)
+server Unknown and clear the server's connection pool if the TopologyType is
```

Comment:
Nit: This makes it sound like we also don't clear the connection pool in LB mode. Can we add a second sentence right after this:

> If the TopologyType is LoadBalanced, drivers MUST NOT mark the server as Unknown for authentication failures but MUST clear the connection pool for the serviceId of the connection.